vllm/sampling_params.py

```diff
@@ -452,6 +484,10 @@ def sampling_type(self) -> SamplingType:
     def all_stop_token_ids(self) -> Set[int]:
         return self._all_stop_token_ids
 
+    @property
+    def bad_words_token_ids(self) -> List[List[int]]:
```

Comment:
Maybe add a comment here to say for internal use only (since we may remove in a future implementation).